cmd/tetra/policyfilter/policyfilter.go

```diff
@@ -28,11 +28,28 @@ func New() *cobra.Command {
 		dumpCmd(),
 		addCommand(),
 		cgroupGetIDCommand(),
+		dumpDebugCmd(),
 	)
 
 	return ret
 }
 
+func dumpDebugCmd() *cobra.Command {
+	mapFname := filepath.Join(defaults.DefaultMapRoot, defaults.DefaultMapPrefix, policyfilter.CgrpNsMapName)
+	ret := &cobra.Command{
+		Use:   "dumpcgrp",
+		Short: "dump cgroup ID to namespace state",
```

Comment:
Cgroup IDs can clash, maybe add node here to give more context as we may not need to store it in map I think

Reply:
Because this is a tetra command I think its OK to omit the node name. When I add the gRPC command that will have the node.